breathecode/media/views.py

```diff
@@ -420,3 +424,33 @@ def get(self, request, media_id=None, media_slug=None):
             resource[header] = response.headers[header]
 
         return resource
+
+class ResolutionView(APIView):
+    @capable_of('read_media')
+    def get(self, request, media_id=None, academy_id=None):
+
+        if media_id:
+            items = MediaResolution.objects.filter(media_id=media_id)
+
+            if not items:
+                raise ValidationException('Resolutions for this media were not found', code=404, slug='media-resolutions-not-found')
+
+        items = MediaResolution.objects.filter(media_id=media_id)
+        sort = request.GET.get('sort')
+        if sort:
+            items = items.order_by(sort)
+
+        serializer = GetResolutionSerializer(items, many=True)
+
+        return Response(serializer.data, status=status.HTTP_200_OK)
+            
+    @capable_of('crud_media')
+    def delete(self, request, resolution_id=None, academy_id=None):
+
+        data = MediaResolution.objects.filter(id=resolution_id).first()
+        if not data:
+            raise ValidationException('Resolution was not found', code=404, slug='resolution-not-found')
```

Comment:
If all the media resolutions from the same media are remove or all the media parent are removed, the resolution will be remove in google cloud storage

Reply:
A media represent the file that the academy think it had, if two academies upload the same file, in google cloud we just save one, we should do the same with the resolutions